android/src/test/java/com/mparticle/react/IdentityApiTest.java

```diff
@@ -17,24 +17,29 @@
 import org.json.JSONObject;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import java.util.Random;
 
 import static junit.framework.TestCase.assertEquals;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
+@RunWith(MockitoJUnitRunner.class)
 public class IdentityApiTest {
     MParticleModule identityApi;
     Random random = new Random();
 
     @Before
     public void before() {
+        MockitoAnnotations.openMocks(this);
         MParticle.setInstance(Mockito.mock(MParticle.class));
         Mockito.when(MParticle.getInstance().Identity()).thenReturn(Mockito.mock(IdentityApi.class));
-        Mockito.when(MParticle.getInstance().Identity().getUser(null)).thenReturn(null);
+        Mockito.lenient().when(MParticle.getInstance().Identity().getUser(0L)).thenReturn(null);
```

Comment:
Do we accept nulls? If so we need a separate test for this.

Reply:
`getUser` annotation asks for a Long variable